<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QCustomPlot" name="graphWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>701</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="uartLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>UART</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="receivedDataText">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>110</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="amplitudeDial">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>410</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="frequencyDial">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>410</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="amplitudeLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>530</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Amplitude: </string>
   </property>
  </widget>
  <widget class="QLabel" name="frequencyLabel">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>530</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Frequency:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
